docs/vm-images.md

```diff
@@ -1,10 +1,31 @@
 # Supported and Tested VM Images
 
-* [HPC CentOS 7 VM Image](#hpc-centos-7-vm-image)
-* [Ubuntu](#ubuntu)
+* [HPC Rocky Linux 8](#hpc-rocky-linux-8)
+* [HPC CentOS 7](#hpc-centos-7)
+* [Debian 11](#debian-11)
+* [Ubuntu 20.04 LTS](#ubuntu-2004-lts)
 * [Other Images](#other-images)
 
-## HPC CentOS 7 VM Image
+## HPC Rocky Linux 8
+
+HPC Rocky Linux 8 is the primary supported VM image for HPC workloads on Google Cloud.
+
+Exceptions:
+
+* [htcondor-configure]: Only provides support for the HPC CentOS 7 image.
+* [nfs-server]: Only provides support for CentOS 7 images for the server itself.
+* [DDN-EXAScaler]: The underlying terraform module has a limited set of
+  supported images [documented][exascalerimages] in the exascaler-cloud-terraform
+  github repo. **This requirement only applies to the servers, not the clients.**
+
+| Compute type | Shared Filestore | Lustre Networked Storage | Startup Script | Chrome Remote Desktop |
```

Comment:
change compute type to "client module" and write the name of the modules (exception for Slurm where you should say "schedmd-slurm-gcp-v5*"). 
Also, move this table to before the Exceptions.

Reply:
Done.